index.json

```diff
@@ -177,17 +182,25 @@
     ],
     "keywords": [
       "\\\\GAS",
+      "03",
       "111hz",
       "150Hz",
       "15Hz",
       "166Hz",
       "1S",
+      "2.5W",
+      "2022",
       "20ms",
+      "2500mW",
       "250Hz",
+      "2in",
       "2ms",
-      "3 inch",
+      "2S",
+      "3 Inch",
```

Comment:
Please try to use the same name - either `in` or` inch` - and same capitalization as this is confusing.